menas/ui/components/dataset/addDataset.fragment.xml

```diff
@@ -29,19 +29,43 @@
                         <core:Fragment type="XML" fragmentName="components.schema.selector.schemaSelector"/>
                         <Label text="HDFS raw data folder path"/>
```

Comment:
Can we rename this to something more generic? Alternatively change the text with the switch as well?

Reply:
True, makes sense. Renamed.